build.gradle

```diff
@@ -131,6 +132,8 @@ dependencies {
 
     implementation 'ch.qos.logback:logback-classic:1.2.11'
 
+    implementation 'com.esotericsoftware.yamlbeans:yamlbeans:1.11'
```

Comment:
There is a vulnerability in this library. It was recently removed from Detect's dependencies: https://github.com/blackducksoftware/synopsys-detect/pull/939/files
Let's remove it if it's not needed or come up with an alternative if it is.

Reply:
Oh! I wonder why merging master into the feature branch didn't remove this - container scan changes don't need it. Thanks for catching this. Removed it 👍